src/functionalTest/resources/scenarios/employment/ET-RET-4095-CompleteTask.json

```diff
@@ -92,7 +92,7 @@
                 "case_management_category": "Employment",
                 "work_type_id": "applications",
                 "work_type_label": "Applications",
-                "description": "[Create Referral](/cases/case-details/${[CASE_REFERENCE]}/trigger/createReferral/createReferral1)",
+                "description": "[Review Application](/cases/case-details/${[CASE_REFERENCE]}#Applications)",
```

Comment:
There not a test for these two changes ?

Reply:
The other tests are for Referrals which had a change in name as well as description.
This test is for Applications which only had a change in description and not in the name of the Task.